samlsp/middleware_test.go

```diff
@@ -77,12 +77,18 @@ func (test *MiddlewareTest) SetUpTest(c *C) {
 		},
 		TokenMaxAge: time.Hour * 2,
 	}
+
 	cookieStore := ClientCookies{
 		ServiceProvider: &test.Middleware.ServiceProvider,
 		Name:            "ttt",
 	}
 	test.Middleware.ClientState = &cookieStore
 	test.Middleware.ClientToken = &cookieStore
+	test.Middleware.TokenSigner = &JWTTokenSigner{
+		Key:      x509.MarshalPKCS1PrivateKey(test.Middleware.ServiceProvider.Key),
```

Comment:
Just curious, why is this restricted to PKCS1 only? The RSA key could also be PKCS8.